installers/osie/mirror.go

```diff
@@ -42,3 +48,20 @@ func mustBuildOsieURL() *url.URL {
 	}
 	return u
 }
+
+func buildWorkerParams() {
+	dockerRegistry = getParam("DOCKER_REGISTRY")
+	grpcAuthority = getParam("TINKERBELL_GRPC_AUTHORITY")
+	grpcCertURL = getParam("TINKERBELL_CERT_URL")
+	registryUsername = getParam("REGISTRY_USERNAME")
+	registryPassword = getParam("REGISTRY_PASSWORD")
+	elasticSearchURL = getParam("ELASTIC_SEARCH_URL")
+}
+
+func getParam(key string) string {
+	value := os.Getenv(key)
+	if value == "" {
+		log.Fatal(errors.New("invalid" + key))
```

Comment:
this should be using `pkg/log` based logger. This way `Fatal` sends to rollbar for us.